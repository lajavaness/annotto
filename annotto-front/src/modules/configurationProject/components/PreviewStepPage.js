import React from 'react'

const PreviewStepPage = () => {
	return <div>PreviewStepPage</div>
}

export default PreviewStepPage
