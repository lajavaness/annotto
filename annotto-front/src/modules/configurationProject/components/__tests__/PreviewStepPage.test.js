import { Provider } from 'react-redux'
import { ThemeProvider } from 'styled-components'
import { createStore } from 'redux-dynamic-modules'
import { render } from '@testing-library/react'
import React from 'react'

import theme from '__theme__'

import PreviewStepPage from 'modules/configurationProject/components/PreviewStepPage'

const initialState = {}
const getStore = (state) => createStore({ initialState: state })
const getInstance = (state = initialState) => (
	<Provider store={getStore(state)}>
		<ThemeProvider theme={theme}>
			<PreviewStepPage />
		</ThemeProvider>
	</Provider>
)

describe('PreviewStepPage', () => {
	it('matches snapshot', () => {
		const { asFragment } = render(getInstance())
		expect(asFragment()).toMatchSnapshot()
	})
})
