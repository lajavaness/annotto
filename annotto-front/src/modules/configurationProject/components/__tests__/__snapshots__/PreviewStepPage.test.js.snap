// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviewStepPage matches snapshot 1`] = `
<DocumentFragment>
  <div>
    PreviewStepPage
  </div>
</DocumentFragment>
`;
